src/components/NewAnalysisModal.js

```diff
@@ -0,0 +1,103 @@
+import _ from 'lodash/fp'
+import { Fragment, useState } from 'react'
+import { div, h, hr, img } from 'react-hyperscript-helpers'
+import { spinnerOverlay } from 'src/components/common'
+import ModalDrawer from 'src/components/ModalDrawer'
+import { NewGalaxyModalBase } from 'src/components/NewGalaxyModal'
+import { NewRuntimeModalBase } from 'src/components/NewRuntimeModal'
+import { tools } from 'src/components/notebook-utils'
+import TitleBar from 'src/components/TitleBar'
+import galaxyLogo from 'src/images/galaxy-logo.png'
+import jupyterLogoLong from 'src/images/jupyter-logo-long.png'
+import rstudioLogo from 'src/images/rstudio-logo.svg'
+import colors from 'src/libs/colors'
+import { withErrorReporting } from 'src/libs/error'
+import * as Utils from 'src/libs/utils'
+
+
+export const NewAnalysisModal = Utils.withDisplayName('NewAnalysisModal')(
```

Comment:
In general, would love to remove the `New` prefixes from modal names and more when we won't have an old version…not necessarily in this PR but hopefully sooner rather than later 😊

Reply:
I just kept the convention haha. We can probably remove new from all of them. I'll add it to this refactoring ticket https://broadworkbench.atlassian.net/browse/IA-2807